### PR TITLE
Localization changes for web extension

### PR DESCRIPTION
**Pull request by vedg**
Requirements added here: https://github.com/microsoft/vscode-dev/issues/477

To address string literal issue changes are made directly - link to issue - https://github.com/Microsoft/vscode-nls-dev/issues/5

Testing:

npm run compile-web
tested with portal route -
http://localhost:3000/powerplatform/portal/webpages/563a8e53-6cf7-ec11-82e4-000d3a8b4b81?orgUrl=https://org4e297bda.crm10.dynamics.com&dataSource=Dataverse&schema=portalDataverseoldschema&websiteId=24828659-6cf7-ec11-82e4-000d3a8b4b81&websiteName=Site%205&clientSessionId=20d30dc0-040a-11ed-a9c5-d5d08bd7e3e0&Client=PowerPagesStudio

### CODE REVIEW

File: src/web/client/common/authenticationProvider.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export async function dataverseAuthentication(dataverseOrgURL: string): Promise<
         const session = await vscode.authentication.getSession(PROVIDER_ID, [`${dataverseOrgURL}//.default`, 'offline_access'], { createIfNone: true });
         accessToken = session.accessToken;
     } catch (error) {
-        vscode.window.showErrorMessage(ERRORS.AUTHORIZATION_FAILED);
+        vscode.window.showErrorMessage(localize("microsoft-powerapps-portals.webExtension.init", "Authorization Failed. Please run again to authorize it"));
```
Comment (ashishchoudhary001) [inline src/web/client/common/authenticationProvider.ts:28]: Please check if key is correct 'microsoft-powerapps-portals.webExtension.init'?

**Review by ashishchoudhary001 — CHANGES_REQUESTED** [review]
Doesn't look like 'localize' is implemented correctly. For every string we have the same key?
Please revisit, or let me know in case I am missing something.

File: src/web/client/common/authenticationProvider.ts (hunk 1 above)
Comment (vedg) [reply src/web/client/common/authenticationProvider.ts]: Added unique keys

**Review by ashishchoudhary001 — CHANGES_REQUESTED** [review]
Are you missing some changes? These keys should be part of some .json file where they get localized? Please check CliAcquisition.ts for reference.

**Comment by vedg** [comment]
> 

Added the loc files

**Review by ashishchoudhary001 — APPROVED** [review]

**Pull request merged**